lime_timeseries.py

```diff
@@ -3,7 +3,33 @@
 from lime import explanation
 from lime import lime_base
 import math
+import logging
 
+class MultiTSDomainMapper(explanation.DomainMapper):
```

Comment:
Maybe TSDomainMapper or TimeSeriesDomainMapper? Because it also handles univariate TS now.

Reply:
commit e60b38b2cab3e529af89869ff183a077c5f1a1da